src/loader.ts

```diff
@@ -246,10 +249,10 @@ interface ModuleDefinitionArguments extends Array<any> {
 		if (Array.isArray(configuration) || /* require(mid) */ typeof configuration === 'string') {
 			callback = <RequireCallback> dependencies;
 			dependencies = <string[]> configuration;
-			configuration = {};
+			configuration = undefined;
 		}
 
-		has('loader-configurable') && configure(configuration);
```

Comment:
this was causing `configure` function to be called twice so have set to undefined and only calling conditionally. This will be removed when realigning our `require` function with AMD spec.